docs/geedocs/5.3.0/answer/7160007.html

```diff
@@ -103,32 +97,103 @@ <h5>
               Files are linked in <code>/tmp</code> using symbolic links back to original locations.
             </td>
           </tr>
+          <tr>
+            <td>279</td>
+            <td>Server upgrade resets the Admin Console password</td>
+            <td>
+              When the GEE server is upgraded, either through the <code>install_server.sh</code> script or the RPMs, the Admin Console password will be preserved if the password file is found at the expected location.
+            </td>
+          </tr>
           <tr>
             <td>799</td>
             <td>The GEE Portable Server does not return valid JSON when queried with <code>http://&lt;server&gt;/&lt;map&gt;/query?request=Json</code>.</td>
             <td>
               Added a <code>v</code> parameter to the query; when its value is 2, the GEE Portable Server will return proper JSON.  Currently, it defaults to 1 if not specified.
             </td>
           </tr> 
-           <tr>
+          <tr>
+            <td>1093</td>
+            <td>The <code>geselectpublishroot</code> server command line tool offers no help, and appears to take some action when the <code>--help</code> flag is specified</td>
+            <td>
+              Help has been added through the <code>--help</code> option, or if any invalid parameter is given
+            </td>
+          </tr>
+          <tr>
+            <td>1098</td>
+            <td>TravisCI can pass when compilation fails</td>
+            <td>
+              Add logic to cause compilation failures to be detected and reported
+            </td>
+          </tr>
+          <tr>
+            <td>1104</td>
+            <td>Centos 7 (and maybe others) fails to build OpenGEE. Build seems to fail when building third party parts of Qt. This appears to be an issue with libpng.</td>
+            <td>
+              Fixed by including <code>libpng12</code> in the build script.
+            </td>
+          </tr>
+          <tr>
+            <td>1122</td>
+            <td>Check for PIL (Python Imager Library) returns an error</td>
+            <td>
+              Change how PIL is included.
+            </td>
+          </tr>
+          <tr>
+            <td>1138</td>
```

Comment:
I would leave this one out since it was introduced and fixed in the latest release.